webapp/app.py

```diff
@@ -63,10 +63,57 @@ def results():
     return flask.render_template("careers/results.html", **context)
 
 
-@app.route("/careers/<department>", methods=["GET", "POST"])
+@app.route("/careers/thank-you", methods=["POST"])
```

Comment:
no post to thank you page. It doesn't makes sense. Keepp the POST how it was and then redirect to `/careers/thank-you`